Bottom: f49f42b77b25f3d17c1bde3b76f3aefb9998881d
Top:    5aa5621a18d7eeeca9f321666c6ef6ba28354d79
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-19 20:35:31 +0800

Refresh of vhost_net-split-out-datacopy

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index 886a603..9cef0b2 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -585,7 +585,7 @@ static void handle_tx_zerocopy(struct vhost_net *net, struct socket *sock)
 {
 	struct vhost_net_virtqueue *nvq = &net->vqs[VHOST_NET_VQ_TX];
 	struct vhost_virtqueue *vq = &nvq->vq;
-	unsigned int out, in;
+	unsigned out, in;
 	int head;
 	struct msghdr msg = {
 		.msg_name = NULL,
